Entities/Scene.cpp

```diff
@@ -2904,12 +2936,14 @@ void Scene::UpdatePathFinding()
 // Description:     Calculates and returns the least difficult path between two points on
 //                  the current scene. Takes both distance and materials into account.
 
-float Scene::CalculatePath(const Vector &start, const Vector &end, std::list<Vector> &pathResult, float digStrenght)
+float Scene::CalculatePath(const Vector &start, const Vector &end, std::list<Vector> &pathResult, float digStrength, Activity::Teams team)
 {
     float totalCostResult = -1;
-    if (m_pPathFinder)
+
+    PathFinder* pathfinder = GetPathfinder(team);
```

Comment:
Should be `PathFinder *pathFinder` (though I'd be fine with `PathFinder *pathfinder` too, cause I don't really like the capital F in the class name haha)

Applies below as well